.eslintrc.json

```diff
@@ -41,6 +41,7 @@
         ],
       	"no-console": "warn",
         "no-var": "error",
-        "no-unused-vars": "warn"
+        "no-unused-vars": "warn",
+				"no-prototype-builtins": "warn"
```

Comment:
Weird mix of spaces and tabs in this file

Reply:
Ick you're right.